sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -483,8 +470,6 @@ private[execution] final class LongToUnsafeRowMap(val mm: TaskMemoryManager, cap
    */
   def getValue(key: Long, resultRow: UnsafeRow): UnsafeRow = {
     if (isDense) {
-      numKeyLookups += 1
-      numProbes += 1
```

Comment:
If as your test shows this is the cause of performance regression, we can just revert this and related changes. The change in `HashAggregateExec`, etc. can be kept.

Reply:
If this is proved to cause perf regression, I think it's safer to revert `HashAggregateExec` as well, since they are doing the same thing.

Reply:
+1, like I said in https://github.com/apache/spark/pull/23204/files#r238257371